lib/AST/ASTPrinter.cpp

```diff
@@ -3159,6 +3159,24 @@ static bool usesFeatureExistentialAny(Decl *decl) {
   return false;
 }
 
+static bool usesFeatureImportObjcForwardDeclarations(Decl *decl) {
```

Comment:
Is this being used for anything? I don't see it referenced elsewhere in this PR.

Reply:
Yes, but not obviously -- above in the same file:

```
#define LANGUAGE_FEATURE(FeatureName, SENumber, Description, Option)  \
    if (usesFeature##FeatureName(decl))                               \
      collectRequiredFeature(Feature::FeatureName, operation);
#define SUPPRESSIBLE_LANGUAGE_FEATURE(FeatureName, SENumber, Description, Option)  \
    if (usesFeature##FeatureName(decl))                               \
      collectSuppressibleFeature(Feature::FeatureName, operation);
#include "swift/Basic/Features.def"
  }
};
```
It won't compile without this.

Reply:
Ah, metaprogramming. Thanks for clarifying!